alpha/dotnet-chart/templates/buildconfig.yaml

```diff
@@ -26,13 +26,8 @@ spec:
     type: Source
     sourceStrategy:
       from:
-        kind: ImageStreamTag
-        name: {{ .Values.build.imageStreamTag.name }}
-  {{- if .Values.build.imageStreamTag.useReleaseNamespace }}
-        namespace: {{ .Release.Namespace }}
-  {{- else }}
-        namespace: {{ .Values.build.imageStreamTag.namespace }}
-  {{- end }}
+        kind: DockerImage
```

Comment:
This change is meant to deal with the tag not being available yet, so it is referencing the image directly.

Can we support `DockerImage` as an alternative to `ImageStreamTag` instead of a replacement?

It would be nice if we still support the parameters that were available in the previous version, and keep similar parameters/features as the other charts (like the `quarkus-chart`)